todo_app/data/view_model.py

```diff
@@ -0,0 +1,40 @@
+from todo_app.data.item import Item
+
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def todo_items(self) -> list[Item]:
+        output = []
```

Comment:
Nice!

Not a recommendation, but I often think it's worth seeing the "list comprehension" approach as a comparison - it's much shorter, but up for debate as to whether it makes it easier to follow:

```python
return [item in self._items if item.status == "To do"]
```